js/picture-load-form.js

```diff
@@ -31,66 +37,14 @@ export const openPictureLoadEditor = () => {
     document.addEventListener('keydown', onPictureEditorEscKeydown);
   };
 
-  // Scale picture logic
-  const scaleControlSmaller = document.querySelector('.scale__control--smaller');
-  const scaleControlBigger = document.querySelector('.scale__control--bigger');
-  const scaleControlValue = document.querySelector('.scale__control--value');
-  const previewPicture = document.querySelector('.img-upload__preview');
-  let scaleStep = parseInt(scaleControlValue.value, 10) / 100;
-  previewPicture.style.transform = `scale(${scaleStep})`;
-
-
-  function makeScaleSmaller() {
-    if (scaleControlValue.value !== scaleControlValue.min) {
-      scaleControlBigger.removeAttribute('disabled', true);
-      scaleControlValue.value = `${parseInt(scaleControlValue.value, 10) - 25}%`;
-      scaleStep = parseInt(scaleControlValue.value, 10) / 100;
-      previewPicture.style.transform = `scale(${scaleStep})`;
-
-    } else {
-      scaleControlSmaller.setAttribute('disabled', true);
-    }
-  }
-
-  scaleControlSmaller.addEventListener('click', makeScaleSmaller);
-
-  function makeScaleBigger() {
-    if (scaleControlValue.value !== scaleControlValue.max) {
-      scaleControlSmaller.removeAttribute('disabled', true);
-      scaleControlValue.value = `${parseInt(scaleControlValue.value, 10) + 25}%`;
-      scaleStep = parseInt(scaleControlValue.value, 10) / 100;
-      previewPicture.style.transform = `scale(${scaleStep})`;
-    } else {
-      scaleControlBigger.setAttribute('disabled', true);
-    }
-  }
-
-  scaleControlBigger.addEventListener('click', makeScaleBigger);
-
-  // picture effects changing
-  const pictureEffectsFieldset = document.querySelector('.img-upload__effects');
-  const pictureEffectsList = pictureEffectsFieldset.querySelector('.effects__list');
-  const pictureEffectInputChecked = pictureEffectsList.querySelector('[checked]');
-
-  function changeLoadPictureEffect(evt) {
-    const targetPoint = evt.target.closest('.effects__item');
-    const requiredClass = targetPoint.querySelector('.effects__preview').classList[1];
-
-    if (pictureEffectInputChecked) {
-      previewPicture.classList.remove(previewPicture.classList[1]);
-      previewPicture.classList.add(requiredClass);
-    }
-  }
-
-  pictureEffectsList.addEventListener('change', changeLoadPictureEffect);
-
-  // close picture load window
+  // close picture editor
   function closePictureEditor() {
     loadPictureEditor.classList.add('hidden');
     pageBody.classList.remove('modal-open');
     previewPicture.classList.remove(previewPicture.classList[1]);
-    // pristine.reset();
     uploadFileInput.innerHTML = '';
```

Comment:
```suggestion
    uploadFileInput.value = '';
```
у инпута нужно сбрасывать `value`